internal/pkg/build/sources/packer_ext3_linux.go

```diff
@@ -31,6 +33,22 @@ func (p *Ext3Packer) Pack(context.Context) (*types.Bundle, error) {
 	return p.b, nil
 }
 
+func getMaxLoopDevices() int {
```

Comment:
This `getMaxLoopDevices` is repeated in many locations - please instead define it once as an exported/Public function (probably in `pkg/util/loop` is a good place - and then call it from other places it is needed.